tests/integration/transformers/test_base.py

```diff
@@ -132,84 +132,3 @@ def _reverse_transform(self, data):
     })
     pd.testing.assert_frame_equal(expected_transform, transformed)
     pd.testing.assert_frame_equal(reverse, data)
-
-
-def test_dummy_transformer_multi_column_input():
```

Comment:
This test is no longer supported, and since the entire purpose of the test is to show that `multi_column_input` works, I didn't fell like it made much sense to update it to conform with the new code (it would end up just repeat testing what the test above already does).